files/galaxy/tpv/roles.yml

```
@@ -6,3 +6,16 @@ roles:
   storage-test*:
     params:
       object_store_id: "s3_netapp01"
+
+  rstudio-poweruser*:
+    rules:
+      - id: rstudio_poweruser
+        if: |
+          'rstudio' in tool.id
```

I don't know why another tool with rstudio in its id may pop up, but looking at how many tools we have installed with `notebook` or `jupyter` in their ids I believe the change below buys us future sanity. Sorry to annoy you with the little things.

```suggestion
        if: 'interactive_tool_rstudio' == tool.id
```

---

without the pipe character I get a yaml error

---

Let the yaml character be then haha

---

https://sidneyliebrand.medium.com/the-greatnesses-and-gotchas-of-yaml-5e3377ef0c55

see under "Multiline strings? YAML’s got your back!"